nautobot/circuits/models.py

```diff
@@ -327,6 +337,21 @@ def clean(self):
             raise ValidationError("A circuit termination must attach to either a site or a provider network.")
         if self.site and self.provider_network:
             raise ValidationError("A circuit termination cannot attach to both a site and a provider network.")
+        # Iff a site is defined, a location *may* also be defined.
```

Comment:
```suggestion
        # If a site is defined a location *may* also be defined.
```

Reply:
`Iff` was intentional, as shorthand for `If and only if`, but I recognize that's not necessarily obvious. 
```suggestion
        # If and only if a site is defined, a location *may* also be defined.
```